src/pfe/file-watcher/server/src/projects/ShellExtensionProject.ts

```diff
@@ -86,6 +86,7 @@ export class ShellExtensionProject implements IExtensionProject {
 
     supportedType: string;
     // defaultIgnoredPath: string[] = ["*/*"];
+    builtByExtension: boolean = true;
```

Comment:
thanks for this change, could you remove the commented out line above as well?

Reply:
updated